src/store/index.js

```diff
@@ -11,188 +11,225 @@ export default new Vuex.Store({
-        category: "Java",
-        limit: "2020-08-14",
-        completed: false
+        category: 'Java',
+        limit: '2020-08-14',
+        completed: false,
       },
       {
         id: 1,
-        title: "Vue Todo",
+        title: 'Vue Todo',
         priority: 2,
-        category: "Vue",
-        limit: "2019-08-14",
-        completed: false
+        category: 'Vue',
+        limit: '2019-08-14',
+        completed: false,
       },
       {
         id: 2,
-        title: "React course",
+        title: 'React course',
         priority: 1,
-        category: "React",
-        limit: "2019-12-20",
-        completed: false
+        category: 'React',
+        limit: '2019-12-20',
+        completed: false,
       },
       {
         id: 3,
-        title: "React Calendar",
+        title: 'React Calendar',
         priority: 1,
-        category: "React",
-        limit: "2019-11-30",
-        completed: false
+        category: 'React',
+        limit: '2019-11-30',
+        completed: false,
       },
       {
         id: 4,
-        title: "CS50",
+        title: 'CS50',
         priority: 0,
-        category: "YouTube",
-        limit: "2020-09-12",
-        completed: false
+        category: 'YouTube',
+        limit: '2020-09-12',
+        completed: false,
       },
       {
         id: 5,
-        title: "Illustrator crashcourse",
+        title: 'Illustrator crashcourse',
         priority: 0,
-        category: "Adobe",
-        limit: "2020-08-22",
-        completed: false
+        category: 'Adobe',
+        limit: '2020-08-22',
+        completed: false,
       },
       {
         id: 6,
-        title: "InDesign",
+        title: 'InDesign',
         priority: 0,
-        category: "Adobe",
-        limit: "2021-09-02",
-        completed: false
-      }
-    ]
+        category: 'Adobe',
+        limit: '2021-09-02',
+        completed: false,
+      },
+    ],
   },
   getters: {
     todoLength: state => {
-      return state.todos.length
+      return state.todos.length;
     },
     todoCompleted: state => {
-      return state.todos.filter(x => x.completed).length
+      return state.todos.filter(x => x.completed).length;
     },
     completionRate: (state, getters) => {
-      return getters.todoLength === 0 ? i18n.t('status.noTasks') :
-        Math.round((getters.todoCompleted / getters.todoLength) * 100) + "%"
+      return getters.todoLength === 0
+        ? i18n.t('status.noTasks')
+        : Math.round((getters.todoCompleted / getters.todoLength) * 100) + '%';
     },
     dateToday(state) {
-      let today = state.today
-      const yyyy = today.getFullYear()
-      let mm = today.getMonth() + 1
-      let dd = today.getDate()
+      // this function seems like it doesn't need to use state,
+      // why not just use new Date() (or Date.now() which is more readable) every time?
+      // you can also use a library for common tasks like this,
+      // check out date-fns https://date-fns.org/
```

Comment:
Is it okay to leave this vanilla JS? I sort of want to prove that I can do it vanilla, before opting for libraries.

Reply:
sure, you can do whatever you want 😛 just FYI, for when you're working for real, better to use a library that's already maintained and tested for all kinds of edge cases (less code for you to maintain)
also date-fns is a good library to know